### DISCUSSION TITLE
make it work with ollama

### DISCUSSION THREAD

**Discussion by secretgspot** [AmberSahdev/Open-Interface#19] [Ideas]
as per subject so I don't have to pay .20c per request

**Comment by genidma** [comment]
- Why just ollama @secretgspot Just wondering?
- Doing a service level scan and I think there are other options also? Sample here via FreeCodeCamp (Sharing does not equal endorsement) [https://www.freecodecamp.org/news/how-to-run-open-source-llms-on-your-own-computer-using-ollama/](https://www.freecodecamp.org/news/how-to-run-open-source-llms-on-your-own-computer-using-ollama/)

- Also, I think the Readme suggests costs ranging from 0.05 to 0.20 c. As well, a clause how pricing could come down (something to do with agents)

Cheers!
